@ndlib/gatsby-theme-marble/src/components/Shared/ManifestCard/ManifestCardChildren/index.js

```diff
@@ -55,8 +55,8 @@ export const findMetadata = (manifest, options) => {
 }
 
 export const findCreator = (item, parentProps) => {
-  if (parentProps.highlight && parentProps.highlight.creator) {
-    return parentProps.highlight.creator[0]
+  if (parentProps.highlight && parentProps.highlight['creator.folded']) {
```

Comment:
Should we just make this a check with `typy`? 
```if (typy(parentProps, 'highlight.creator.folded').isArray) {```

Reply:
let me try I was not sure about how typy was going to handle that being a property not an object.

Reply:
this is not working.  it does not seem to id keys with dots in them.